project_resource_calendar/wizard/weekly_report_wizard.py

```diff
@@ -0,0 +1,81 @@
+from datetime import datetime, timedelta
+
+from odoo import models, fields, api
+from odoo.tools import DEFAULT_SERVER_DATE_FORMAT as DATE_FORMAT
+
+
+class WeeklypReportWizard(models.TransientModel):
+    _name = 'weekly.report.wizard'
+
+    date_start = fields.Date(string="Start Date", required=True, default=fields.Date.today)
+    date_end = fields.Date(string="End Date", required=True, default=fields.Date.today)
+    room_id = fields.Many2one(
+        string='Room',
+        comodel_name='resource.calendar.room',
+        ondelete='set null',
+        required=True,
+    )
+    recurrency = fields.Boolean(
+        string='Recurrency',
+        default=True,
+        required=True,
```

Comment:
why required: if it is not true so it is false

Reply:
it is required, on the interface this is true, but on the api it is required

Reply:
I will remove it

Reply:
Done